analytics/src/analytics/metrics/burnup.py

```diff
@@ -184,15 +184,9 @@ def _isolate_data_for_this_sprint(self) -> pd.DataFrame:
 
     #     """
```

Comment:
`ruff` made these changes. This is a good reminder not to leave large code blocks commented out like this, its "unsafe"